packages/socks-proxy-agent/package.json

```diff
@@ -109,7 +109,7 @@
   "dependencies": {
     "agent-base": "^7.0.2",
     "debug": "^4.3.4",
-    "socks": "^2.7.1"
+    "socks": "^2.7.3"
```

Comment:
this change isn't required because the constraint allows for `2.7.3` to be installed - while it might seem like a good idea to update the min. version, it means if there is an issue with 2.7.3 then people have to resort to workarounds like `resolutions` to potentially unblock themselves.

It's better to leave the constraint as is, and people can just run `npm update socks` to get the new version.

Reply:
reverted the last commit